static/beta/stage/navigation/trust-navigation.json

```diff
@@ -0,0 +1,11 @@
+{
+    "id": "trust",
+    "title": "Trusted Content",
+    "navItems": [
+        {
+            "title": "Trusted Content",
+            "appId": "trust",
+            "href": "/trust"
```

Comment:
Again this URL does not seems correct. The link href should include the module route.